libV2/index.js

```diff
@@ -104,7 +104,7 @@ module.exports = {
           }
           catch (error) {
             console.error(error);
-            break;
+            throw error;
```

Comment:
@thim81 This error handling at request level is intentional. Reason is that we don't want to fail the entire collection generation process if a certain request fails due to an issue, making sure other requests are converted correctly.

Reply:
Understood, I reverted the throw.